### ISSUE TITLE
Missing data

### ISSUE THREAD

**Issue by NaVi-start** [how2sign/how2sign.github.io#23]
How2Sign has less 2D data than text data. For example, the test set of How2Sign has 2343 2D videos, but 2357 text data. The Val set and training set also have this problem.

**Comment by abdullaharifx** [comment]
There are issues in the test set
Some videos do not have any sign performed by the signer
However, there is a sentence/phrase/word written in the csv against the respective ID of the video
A few names are given:
"fzrJBu2qsM8_11-8-rgb_front
g3Cc_1-V31U_23-3-rgb_front
FzoUVr98JmQ_0-8-rgb_front"

Please ack if someone faces the same issue.

**Comment by TuGuobin** [comment]
I also encountered the same problem! The data I downloaded is rgb_front_clips. Among them, the train (31048 videos / 31165 texts), val (1739 / 1741) and test (2343 / 2357) data do not match. Is there any available solution?
